esp-hal/src/spi/slave.rs

```diff
@@ -22,8 +22,8 @@
 //! # use esp_hal::dma::Dma;
 //! # use esp_hal::gpio::Io;
 //! let dma = Dma::new(peripherals.DMA);
-#![cfg_attr(esp32s2, doc = "let dma_channel = dma.spi2channel;")]
-#![cfg_attr(not(esp32s2), doc = "let dma_channel = dma.channel0;")]
+#![cfg_attr(pdma, doc = "let dma_channel = dma.spi2channel;")]
```

Comment:
not(gdma) ?

Reply:
Do you expect future devices to return to per-peripheral DMA channels?

Reply:
ah no .... I read `gdma` for some reason here .... time for another coffee then